ee/debug/checkups/download_directory.go

```diff
@@ -0,0 +1,117 @@
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+type DownloadDirectory struct {
+	status  Status
+	summary string
+	files   []string
+}
+
+func (c *DownloadDirectory) Name() string {
+	return "Download directory contents"
+}
+
+func (c *DownloadDirectory) Run(_ context.Context, extraFH io.Writer) error {
+	downloadDir := getDownloadDir()
+	if downloadDir == "" {
+		return fmt.Errorf("no default download directory")
+	}
+
+	err := filepath.Walk(downloadDir, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if !info.IsDir() && isKolideInstaller(info.Name()) {
+			c.files = append(c.files, path)
```

Comment:
Sorry, I completely missed documenting this on the original issue -- could we also include the time each file was created, so we know when the user performed the download? Can grab it here with `info.ModTime()`, I think.

Reply:
added support for Mod time